meta.yaml

```diff
@@ -5,10 +5,10 @@ package:
   version: '{{ environ.get("GIT_DESCRIBE_TAG", data.get("version")) }}'
 
 source:
-  path: .
+#  path: .
 # ideally we want this git_url path, but it messes with conda publish
 # where it builds only the sourced url and not local/branch changes
-# git_url: https://github.com/NOAA-GFDL/fre-cli
+  git_url: https://github.com/NOAA-GFDL/fre-cli.git
```

Comment:
wait are you telling me the problem was the lack of `.git` ??

Reply:
It might've been. I was following an example I saw somewhere and they included the `.git`

Reply:
OK, if we end up with a mysterious problem at the `publish_conda` step, we'll know @bcc2761's old comment still applies.